contents/handbook/small-teams/experimentation/objectives.mdx

```diff
@@ -10,6 +10,29 @@
 * **Rationale:**
  * Make sure that experimentation is a first class feature, Posthog users are excited to use alongside the rest of the analytics features
 
+
+----- Suggested replacement for Experimentation focus
+
+Objective: Ensure that feature flags can mature with users
+
+* **Key Result:** X users on permission and alerting features
```

Comment:
```suggestion
* **Key Result:** 5 happy teams on permission and alerting features
```

Let's pick an actual number. I suggest 5 "happy" ones since:

* It means the feature is useful
* It means you're working with a few teams so we don't over-fit to just one